src/main/java/siri/Task.java

```diff
@@ -0,0 +1,72 @@
+package siri;
+/**
+ * Class for the individual items of the task list, with int done to be indicative of whether the task had
+ * been completed and String item to be the name of the task.
+ */
+
+class Task {
+    protected boolean isDone;
+    protected String item;
+
+    /**
+     * Constructor for Task class.
+     *
+     * @param item a String to description of the Task.
+     * @param done an int to indicate whether the item is done. 0 indicates not completed and 1 indicates completed.
+     */
+    Task(String item, boolean isDone) {
+        this.isDone = isDone;
+        this.item = item;
+    }
+
+    /**
+     * Returns a symbol the doneness of the task.
+     * If task is done, return 'X', else return empty space to indicate task not done.
+     *
+     * @param done integer to indicate whether done or undone. 1 represents the done and 0 represents undone.
+     * @return a string " " to indicate the task had not been completed and "X" to indicate the task had be completed.
+     */
+    private String doneSymbol(boolean isDone) {
```

Comment:
Perhaps this method can be renamed into something that hints at more of a getter? e.g. getDoneSymbol()